package.json

```diff
@@ -32,7 +32,17 @@
     "cypress:update-snapshots": "TZ=UTC cypress run --component --spec cypress/visual/**/*.cy.js --env type=base --config screenshotsFolder=cypress/snapshots/base"
   },
   "main": "dist/ncvuecomponents.js",
-  "module": "dist/index.module.js",
+  "exports": {
+    ".": {
+      "import": "./dist/index.mjs",
+      "require": "./dist/index.js"
+    },
+    "./dist/install.js": "./dist/install.js",
+    "./dist/Components/*.js": "./dist/Components/*.js",
+    "./dist/Directives/*.js": "./dist/Directives/*.js",
+    "./dist/Functions/*.js": "./dist/Functions/*.js",
+    "./dist/Mixins/*.js": "./dist/Mixins/*.js"
```

Comment:
```suggestion
    "./dist/": "./dist/",
```

Reply:
Not sure about this. When we build also the other entry points using vite we will get shared chunks which we probably do not want to export.
But for the moment I guess we can do it that way.

Reply:
I think it's more flexible yes